.github/workflows/docker-develop.yml

```diff
@@ -14,13 +17,40 @@ jobs:
     env:
       DOCKER_BUILDKIT: 1
     steps:
-      -   name: Checkout repository
-          uses: actions/checkout@v2
-
-      -   name: Publish to Docker Hub
-          uses: docker/build-push-action@92e71463491f2d026a477188b8ad3a0fdd9d672c
-          with:
-            repository: iotaledger/goshimmer
-            username: '${{ secrets.IOTALEDGER_HUB_DOCKER_LOGIN }}'
-            password: '${{ secrets.IOTALEDGER_HUB_DOCKER_PASSWORD }}'
-            tags: develop
+      - name: Checkout repository
+        uses: actions/checkout@v2.3.4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1.3.0
+
+      - name: Cache Docker layers
+        uses: actions/cache@v2.1.6
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1.9.0
+        with:
+          username: '${{ secrets.IOTALEDGER_HUB_DOCKER_LOGIN }}'
+          password: '${{ secrets.IOTALEDGER_HUB_DOCKER_PASSWORD }}'
+
+      - name: Publish to Docker Hub
+        uses: docker/build-push-action@v2.5.0
+        with:
+          build-args: |
+            mount='id=go_build_cache_mount,type=bind,source=/tmp/foo'
```

Comment:
`source=/tmp/foo` does it really not matter? Shouldn't that be maybe `/tmp/.buildx-cache`?

Reply:
Sorry, debug code. I will clean this up